spec/spec_helper.rb

```diff
@@ -4,7 +4,7 @@
 
 RSpec.configure do |config|
   config.formatter = :documentation
-  config.hiera_config = 'spec/fixtures/hiera/hiera.yaml'
+  config.hiera_config = 'spec/hiera.yaml'
```

Comment:
seems that we need to use hiera v3 for spec testing, which is why this points to a similar but different file from the `hiera.yml` in the root of the module.